src/components/Button/__snapshots__/Button.test.tsx.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`<Button /> Default story renders snapshot 1`] = `
 <button
-  class="clickable-style clickable-style--lg clickable-style--secondary clickable-style--brand button button--secondary"
+  class="button button--lg button--secondary button--brand"
```

Comment:
Note for reviewers: if **any** snapshot has a tag changing, this is probably a mistake! The changes should almost entirely be replacing `"chickable-style"` with either `"button"` or `"link"` for now.

Reply:
thanks for the heads up! was going to scroll thru this file tbh